### DISCUSSION TITLE
BPD Brand Kit Strategy

### DISCUSSION THREAD

**Discussion by sandramsc** [BlackPythonDevs/blackpythondevs#54] [Ideas]
As BPD continues to grow and establish its presence within the Python community, we need to think about presenting a cohesive and professional brand identity. A brand kit serves as a comprehensive guide for maintaining consistency in our visual and verbal communications, thereby strengthening our communities' recognition and credibility. 

This proposal outlines the core components of our brand kit and the steps necessary to develop it effectively (or at least an initial working prototype).

For now the idea is to start by creating a basic brand kit that can be enhanced with time by community contributions.

**Core Components:**

1. **Logo:**
   - Develop a visually appealing and scalable logo that embodies the essence of our community.
   - Ensure versatility to maintain clarity and legibility across various platforms and sizes.

2. **Color Palette:**
   - Select a harmonious palette of colors that reflects our communities theme and personality (as a starter we can use the colors that are on the current website).
   - Provide color codes (RGB, HEX, CMYK) for easy reference and consistency in implementation.

3. **Typography:**
   - Define a set of fonts that align with our project's identity and enhance readability.
   - Specify font families for headings, subheadings, and body text, along with styling guidelines.

4. **Imagery Guidelines:**
   - Establish guidelines for selecting and using images or graphics that resonate with our project's aesthetic and messaging (these could be images we create our self i.e a mascot).
   - Include preferred image styles, formats, and sources to maintain visual coherence.

5. **Usage Guidelines:**
   - Create detailed instructions on how to use and apply our brand elements consistently across all project materials.
   - Include guidelines for logo placement, color usage, typography, and image (mascot if we chose to have one) usage to maintain brand integrity.

6. **Brand Assets:**
   - Compile high-resolution versions of our logo, color swatches, and typography files for easy access and use by community contributors/sponsors at events/conferences etc.

7. **Brand Story:**
   - Craft a concise narrative that communicates our communities mission, values, and goals.
   - Share our story to foster understanding and connection with our audience and community.

8. **Legal Information:**
   - Include any necessary legal requirements or restrictions related to the use of our brand elements, such as copyright notices or trademark information.

As for hosting the brand kit, both a website and GitHub can be suitable options:

 Website: Hosting the brand kit on the community website provides easy access for internal and external stakeholders. It can be organized within a dedicated section of the website for easy navigation.

Here's an expanded workflow including the tools and software:
   
1. **Concept Development**:
   - Tools: Sketching tools, mood board platforms (Pinterest, Miro).
   
2. **Design and Creation**:
   - Logo Design:
     - Tools: Adobe Illustrator, Sketch, Canva, Figma.
   - Color Palette:
     - Tools: Adobe Color, Coolors, Color Hunt.
   - Typography:
     - Tools: Adobe Fonts, Google Fonts.
   - Imagery:
     - Tools: Adobe Photoshop, Adobe Stock (or a tool that we could use to design a mascot or imagery that we want to associate with our brand?).

3. **Finalization**:
   - Tools: (free) Graphic design software for final adjustments and refinements.
   
4. **Documentation**:
   - Tools: Google Docs, GitHub Wiki for compiling usage guidelines and brand assets.

This is an open discussion so please add your ideas, suggestions, feedback so we can make this a success together!

**Comment by kjaymiller** [comment]
we've captured some ideas in https://github.com/BlackPythonDevs/.maintainers/discussions/13

There are a lot of thoughts but nothing set in stone.

**Comment by kjaymiller** [comment]
## Logo:

Right now the logo is our name in the [Modern Society](https://www.dafontfree.io/modern-society-free-font/) in UPPERCASE.

On the website the words are stacked, but it is possible to use the words horizontally as well. 

You can find PNGs of the lettering in [this folder on google drive](https://drive.google.com/drive/folders/1OBqQDxTHmL0qDnYRWG39Z-hXh1CKkXlO?usp=sharing)

I have some variations of the lettering (color choices and other formats)
